packages/htmlbars-compiler/lib/compiler.js

```diff
@@ -1,6 +1,7 @@
 /*jshint evil:true*/
 import { preprocess } from "../htmlbars-syntax/parser";
 import TemplateCompiler from "./template-compiler";
+import { wrap } from "../htmlbars-runtime/hooks";
```

Comment:
shouldn't be importing from `htmlbars-runtime` from within `htmlbars-compiler` for anything other than tests (the runtime is not available within Ember for example).  If the wrap function needs to be shared and available to compiler (and ultimately to the consuming library) it should be in `htmlbars-utils`.